nservicebus/upgrades/8to9/index.md

```diff
@@ -1,18 +1,103 @@
+The method used to determine if [immediate dispatch](/nservicebus/messaging/send-a-message.md#dispatching-a-message-immediately) has been requested for a message has been renamed.
+
+snippet: core-8to9-immediate-dispatch
+
+## IManageUnitsOfWork has been removed
+
+The `IManageUnitsOfWork` API has been removed. Instead, a [pipeline behavior should be used to implement custom units of work](/nservicebus/pipeline/unit-of-work.md#implementing-custom-unit-of-work).
+
+snippet: core-8to9-uow
+
+## API to override machine name has changed
+
+The `RuntimeEnvironment.MachineNameAction` API [Override the machine name](/nservicebus/hosting/override-machine-name.md) has been removed. The replacement API is the `HostInfoSettings.UsingHostName` method.
+
+snippet: core-8to9-machinename
+
+## API to set additional audit metadata has changed
+
+The API to [set additional audit metadata](/nservicebus/operations/auditing.md#additional-audit-information) has been changed.
+
+snippet: core-8to9-audit-metadata
+
+## Dependency registration access in features renamed
+
+The property used to access container registrations in features has been renamed from `Container` to `Services`.
+
+snippet: core-8to9-di-features
+
+## Service collection extensions for backwards compatibility removed
+
+Service collection extensions to ease [the transition to Microsoft DI abstractions](/nservicebus/upgrades/7to8/dependency-injection.md#registercomponents-changes) have been removed. It is now required to use the registration APIs added in NServiceBus 8.
+
+snippet: core-8to9-di-shims
+
+## Making features depend on message driven subscriptions
+
+The API to make features depend on [message driven subscriptions](/nservicebus/messaging/publish-subscribe/#mechanics-message-driven-persistence-based) being in use has changed.
+
+snippet: core-8to9-depend-on-subscriptions
+
+## Endpoint addresses
+
+### Access to logical addresses in features
+
+Since endpoint addresses are translated to transport-specific ones later during endpoint startup, addresses are defined using a transport-agnostic `QueueAddress` type. The addresses can be accessed via the `FeatureConfigurationContext`:
+
+snippet: core-8to9-adresses-features
+
+### Access to the physical receive addresses at runtime
+
+Inject the `ReceiveAddresses` type to access the endpoint receive addresses at runtime.
+
+snippet: core-8to9-adresses-runtime
+
+### Dynamic address translation
+
+Inject the new `ITransportAddressResolver` to translate `QueueAddress`'s to a physical transport addresses at runtime.
+
+snippet: core-8to9-adresses-translation
+
+## Restricted type argument for .UsePersistence<T>
+
+https://github.com/Particular/NServiceBus/compare/release-8.1...master#diff-2c08aef8335f8f17ba3dc362fe939f8a5bdddde4f411d06a067882ac204fa43eL854
+
+Proposed action: Exclude since it doesn't impact users
+
+## Exceptions no longer marked as serializable
```

Comment:
I think this will be okay to exclude. From my understanding you only really cared about exceptions being serializable if you needed to transport an exception across an AppDomain boundary.